ui/admin/app/templates/scopes/scope.hbs

```diff
@@ -33,218 +33,314 @@
+                <Hds::Dropdown @enableCollisionDetection={{true}} as |dd|>
+                  <dd.ToggleIcon
+                    @icon='layers'
+                    @text={{t
+                      'titles.feature-edition'
+                      edition=this.featureEdition.edition
+                    }}
+                  />
+
+                  <dd.Title @text={{t 'actions.toggle-feature-edition'}} />
+                  {{#each this.featureEdition.editions as |edition|}}
+                    <dd.Radio
+                      @value={{edition}}
+                      checked={{eq this.featureEdition.edition edition}}
+                      {{on
+                        'change'
+                        (fn this.application.toggleEdition edition)
+                      }}
+                    >
+                      {{edition}}
+                    </dd.Radio>
+                  {{/each}}
+                  <dd.Separator />
+
+                  <dd.Title @text={{t 'actions.toggle-licensed-features'}} />
+                  {{#each this.featureEdition.licensedFeatures as |feature|}}
+                    <dd.Checkbox
+                      @value={{feature}}
+                      checked={{feature-flag feature}}
+                      {{on
+                        'change'
+                        (fn this.application.toggleFeature feature)
+                      }}
+                    >
+                      {{feature}}
+                    </dd.Checkbox>
+                  {{/each}}
+                </Hds::Dropdown>
+              {{/if}}
+
+              <Hds::Dropdown
+                @enableCollisionDetection={{true}}
+                data-test-side-nav-user-dropdown
+                as |dd|
+              >
+                <dd.ToggleIcon @icon='user' @text={{t 'titles.user-menu'}} />
+
+                <dd.Interactive @route='account.change-password'>{{t
+                    'actions.change-password'
+                  }}</dd.Interactive>
+                <dd.Separator />
+
+                <dd.Interactive
+                  {{on 'click' this.application.invalidateSession}}
+                >{{t 'actions.deauthenticate'}}</dd.Interactive>
+                <dd.Separator />
+
+                <dd.Title @text={{t 'actions.toggle-color-theme'}} />
+                {{#each this.application.themes as |theme|}}
+                  <dd.Radio
+                    @value={{theme.value}}
+                    checked={{eq this.session.data.theme theme.value}}
+                    {{on
+                      'change'
+                      (fn this.application.toggleTheme theme.value)
+                    }}
+                  >
+                    {{t (concat 'themes.' theme.label)}}
+                  </dd.Radio>
+                {{/each}}
+              </Hds::Dropdown>
+            {{/if}}
+          </:actions>
+        </Hds::SideNav::Header>
+      </:header>
+      <:body>
+        <Hds::SideNav::Portal::Target />
```

Comment:
Why did we portal this in when it's also in the same template?

Could we not have just used `Hds::SideNav::List`?

Reply:
yup good point, ill take a look at using the list component directly.